ggml-quants.c

```diff
@@ -39,7 +39,7 @@ inline static int32_t vaddvq_s32(int32x4_t v) {
 #ifdef __wasm_simd128__
 #include <wasm_simd128.h>
 #else
-#ifdef __POWER9_VECTOR__
+#ifdef __powerpc64__
```

Comment:
Should this be:

```suggestion
#if defined(__POWER9_VECTOR__) || defined(__powerpc64__)
```

I'm not familiar with ppc64